packages/office-addin-manifest/src/cli.ts

```diff
@@ -28,7 +28,6 @@ commander
 commander
   .command("export")
   .option("-m, --manifest <manfest-path>", "Specify the location of the manifest file.  Default is './manifest.json'")
-  .option("-a, --assets <assets-folder-path>", "Specify the location of the assets folder.  Default is './assets'")
```

Comment:
Why don't we need to specify the assets folder anymore? Did the design change?
Also, please add this command to the README of this package.

Reply:
I had understood that certain icon needed to be a part of the zip file.  However now that I'm actually publishing a zip I can see that it's not required.  On top of that, the ones they need are referenced in the json manifest and these may not be the right ones.  On top of that the register command only passes in the manifest file and I don't have an assets directory to reference and so it wouldn't work unless the assets were in a specific location.

I'm going to switch to using the paths in the manifest (which requires manifest manipulation work) and optionally include them in the zip file . . . no need to specify them directly.